README.md

```diff
@@ -8,15 +8,17 @@ A demo showing off [Isolated Web Apps](https://github.com/WICG/isolated-web-apps
 - [Controlled Frame](https://github.com/WICG/controlled-frame)
 - [Borderless display mode](https://github.com/WICG/manifest-incubations/blob/gh-pages/borderless-explainer.md)
 - [Multiscreen capture with auto-permission](https://github.com/screen-share/capture-all-screens)
+- Unrestricted [Protocol
+  Handlers](https://developer.mozilla.org/en-US/docs/Web/Progressive_web_apps/Manifest/Reference/protocol_handlers)
+  & [Launch
+  Handler](https://developer.mozilla.org/en-US/docs/Web/API/Launch_Handler_API)
+  (the links below require the app to be installed and at least Chrome 140):
+  - [Click me](cf://?text=Lucky&color=peachpuff) if you're feeling lucky
```

Comment:
please describe what the user can expect when clicking on the link.

Reply:
Done